tests/CustomerTest.php

```diff
@@ -137,30 +137,43 @@ public function testCanUpdateSubscription()
         $this->stubRequest(
             'post',
             '/v1/customers/' . $customer->id . '/subscription',
-            array("plan" => "plan")
+            array("plan" => "plan"),
+            array(
+                "object" => "subscription",
+                "id" => "sub_foo"
+            )
         );
-        $customer->updateSubscription(array("plan" => "plan"));
+        $resource = $customer->updateSubscription(array("plan" => "plan"));
+        $this->assertSame("Stripe\\Subscription", get_class($resource));
+        $this->assertSame("sub_foo", $customer->subscription->id);
```

Comment:
This works but it should not be necessary if the lib or stripe-mock did the right thing in the first place right?

Reply:
nevermind this is stubbed hence why it did not do what I expected it to do